client/src/components/CardModal/Activities/ItemComment.module.scss

```diff
@@ -38,6 +38,12 @@
 
   .title {
     padding-bottom: 4px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    position: sticky;
+    top: -1em;
```

Comment:
use 1em because of this

modal has 1em padding

![image](https://github.com/user-attachments/assets/5db67169-9fd3-44ec-a1ea-4ef49f7b6064)